services/autoscaler-adapter/claudie_provider/claudie_provider.go

```diff
@@ -209,7 +210,7 @@ func (c *ClaudieCloudProvider) Refresh(_ context.Context, req *protos.RefreshReq
 func (c *ClaudieCloudProvider) refresh() error {
 	log.Info().Msgf("Refreshing the state")
 	if cluster, err := getClaudieState(c.projectName, c.configCluster.ClusterInfo.Name); err != nil {
-		log.Error().Msgf("error while refreshing a state for the cluster %s : %v", c.configCluster.ClusterInfo.Name, err)
+		log.Err(err).Str("cluster", c.configCluster.ClusterInfo.Name).Msgf("error while refreshing a state for the cluster")
```

Comment:
`error while refreshing a state for the cluster` -> `Error while refreshing a state of the cluster`